command_line_assistant/rendering/renderers.py

```diff
@@ -222,30 +50,68 @@ def __init__(self, plain: bool = False):
         Args:
             plain (bool): Whether to use plain text rendering
         """
-        self._text_renderer: TextRenderer = create_text_renderer(plain=plain)
-        self._warning_renderer: TextRenderer = create_warning_renderer(plain=plain)
-        self._error_renderer: TextRenderer = create_error_renderer(plain=plain)
+        self._plain = plain
+        self._stream_writer: StreamWriter = StreamWriter()
 
-    def success(self, message: str) -> None:
-        """Render a success message.
+    def normal(self, message: str) -> None:
+        """Render a message with a normal color.
 
         Args:
-            message (str): Success message to render
+            message (str): Text to render
         """
-        self._text_renderer.render(message)
+        self._stream_writer.write_line(message)
 
     def warning(self, message: str) -> None:
-        """Render a warning message.
+        """Render a message with a yellow color.
+
+        Args:
+            message: Text to render
+        """
+        if self._plain:
+            self._stream_writer.write_line(message)
+        else:
+            self._stream_writer.write_line("🤔 " + colorize(message, "yellow"))
+
+    def notice(self, message: str) -> None:
+        """Render a message with a bright yellow color.
+
+        Args:
+            message: Text to render
+        """
+        if self._plain:
+            self._stream_writer.write_line(message)
+        else:
+            self._stream_writer.write_line(colorize(message, "bright_yellow"))
```

Comment:
```suggestion
        final_message = message if plain else colorize(message, "bright_yellow")
        self._stream_writer.write_line(final_message)
```

what about something like this?